data/net/minecraft/client/gui/screens/ChatScreen.mapping

```diff
@@ -39,19 +41,21 @@ CLASS net/minecraft/client/gui/screens/ChatScreen
 		ARG 4 partialTick
 	METHOD renderChatPreview (Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/network/chat/Component;FZ)V
 		ARG 1 poseStack
-		ARG 2 preview
+		ARG 2 message
 		ARG 3 alpha
 		ARG 4 canSign
 	METHOD renderChatPreviewHighlights (Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/util/FormattedCharSequence;II)V
 		ARG 1 poseStack
 		ARG 2 sequence
 		ARG 3 minY
 		ARG 4 chatOpacity
+	METHOD requestPreview (Ljava/lang/String;)V
+		ARG 1 message
```

Comment:
Wrong (should be 2 lines up)

Reply:
Forgot to sanitize data. Should be fixed in next commit

Reply:
Not fixed.